app/utils/whitelabel.ts

```diff
@@ -1,31 +1,121 @@
-    return 'https://sfserviceguide.org';
-  }
-  if (isSFFamiliesSite()) {
-    return 'https://sffamilies.sfserviceguide.org/';
-  }
-  return 'https://askdarcel.org';
-};
-
-export {
-  isSFServiceGuideSite, isSFFamiliesSite, getSiteTitle, getSiteUrl,
-};
+// Include new white label here
+type WhiteLabelSiteKey = 'defaultWhiteLabel' | 'SFServiceGuide' | 'SFFamilies';
+
+interface WhiteLabelSite {
+  appImages: {
+    background: string;
+    logoLarge: string;
+    logoSmall: string;
+    algolia: string;
+    mohcdSeal: string;
+  };
+  intercom: boolean;
+  logoLinkDestination: string;
+  navLogoStyle: string;
+  showBanner: boolean;
+  showMobileNav: boolean;
+  showSearch: boolean;
+  siteNavStyle: string;
+  siteUrl: string;
+  title: string;
+  userWay: boolean;
+}
+
+// Include a domain in config.js
+function determineWhiteLabelSite(): WhiteLabelSiteKey {
+  if (window.location.host === config.SFFAMILIES_DOMAIN) return 'SFFamilies';
+  if (window.location.host === config.MOHCD_DOMAIN) return 'SFServiceGuide';
+  return 'defaultWhiteLabel';
+}
+
+const configKey = determineWhiteLabelSite();
+
+/*
+  Specify what is viewed in each white label.
+  A '/' (which is a forward-slash) as a value for logoLinkDestination
+  denotes that the link is internal to the application.
+*/
+const SFFamilies: WhiteLabelSite = {
+  appImages: {
+    background: BackgroundImage,
+    logoLarge: SFFamiliesLogo,
+    logoSmall: SFFamiliesLogo,
+    algolia: SearchByAlgoliaImage,
+    mohcdSeal: SFSeal,
+  },
+  intercom: false,
+  logoLinkDestination: 'https://www.sffamilies.org/',
+  navLogoStyle: styles.navLogoSFFamilies,
+  showBanner: false,
+  showMobileNav: false,
+  showSearch: false,
+  siteNavStyle: styles.siteNavSFFamilies,
+  siteUrl: 'https://sffamilies.sfserviceguide.org/',
+  title: 'SF Families',
+  userWay: true,
+} as const;
+
+const SFServiceGuide: WhiteLabelSite = {
+  appImages: {
+    background: BackgroundImage,
+    logoLarge: SFServiceLogo,
+    logoSmall: SFServiceLogo,
+    algolia: SearchByAlgoliaImage,
+    mohcdSeal: SFSeal,
+  },
+  intercom: true,
+  logoLinkDestination: '/',
+  navLogoStyle: styles.siteNav,
+  showBanner: true,
+  showMobileNav: true,
+  showSearch: true,
+  siteNavStyle: styles.navLogo,
+  siteUrl: 'https://sfserviceguide.org',
+  title: 'SF Service Guide',
+  userWay: false,
+} as const;
+
+const defaultWhiteLabel: WhiteLabelSite = {
+  appImages: {
+    background: BackgroundImage,
+    logoLarge: AskDarcelImage,
+    logoSmall: AskDarcelImage,
+    algolia: SearchByAlgoliaImage,
+    mohcdSeal: SFSeal,
+  },
+  intercom: true,
+  logoLinkDestination: '/',
+  navLogoStyle: styles.siteNav,
+  showBanner: true,
+  showMobileNav: true,
+  showSearch: true,
+  siteNavStyle: styles.navLogo,
+  siteUrl: 'https://askdarcel.org',
+  title: 'AskDarcel',
+  userWay: false,
+} as const;
+
+/*
+  whiteLabel made Readonly to force developer to modify whiteLabel object in this file.
+  Disallow changes at compile time.
+*/
```

Comment:
I'm not sure if this comment provides enough value to justify its existence. It mostly just describes what `Readonly` means in TypeScript without anything that is novel or specific to this codebase. I would argue that the primary purpose of `Readonly` is to do exactly what you've written here, so repeating that here in the form of a comment doesn't add much information, and possibly worse, it draws extra attention for something that is ordinary.

Do you think you could remove this comment?